<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node pkg="nodelet" type="nodelet" name="threaded_manager" args="manager" output="screen">
        <param name="num_worker_threads" value="8"/>
    </node>
    <node pkg="nodelet" type="nodelet" name="electro_optical_nodelet" args="load eeyore_ros/ElectroOpticalNodelet threaded_manager"/>
    <rosparam file="$(find eeyore_ros)/config/electro_optical.yaml"/>

</launch>
